README.md

```diff
@@ -71,13 +71,26 @@ For currently supported operations, see [Commands](##Commands) section.
 
 ### From npm
 
+To install globally:
+
+```sh
+npm install --global @ethersphere/swarm-cli
+```
+
+It may require `sudo` if `npm root --global` is not writable by your
+user.
+
+Or you can install locally:
+
 ```sh
-npm install -g @ethersphere/swarm-cli
+cd [some directory for nodejs files]
+npm install @ethersphere/swarm-cli
+./node_modules/.bin/swarm-cli --help
```

Comment:
didn't work for me, only if I made `npm init` before `npm install`.

Reply:
which version of nodejs?

this works on my NixOS 21.05, nodejs 14.17.4.

it's not a virgin NixOS install, but i don't see anything obvious installed anywhere globally (it's never writable on NixOS), or under my user.

```
$ nix-shell -p nodejs
these paths will be fetched (11.33 MiB download, 56.35 MiB unpacked):
  /nix/store/7sr08in3pxq1izdl97axpy03s8cxd9wi-nodejs-14.17.4
copying path '/nix/store/7sr08in3pxq1izdl97axpy03s8cxd9wi-nodejs-14.17.4' from 'https://cache.nixos.org'...
[nix-shell:~]$ mkdir /tmp/x
[nix-shell:~]$ cd /tmp/x/
[nix-shell:/tmp/x]$ npm install @ethersphere/swarm-cli
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.

> keccak@3.0.1 install /tmp/x/node_modules/keccak
> node-gyp-build || exit 0


> secp256k1@4.0.2 install /tmp/x/node_modules/secp256k1
> node-gyp-build || exit 0

npm WARN saveError ENOENT: no such file or directory, open '/tmp/x/package.json'
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN enoent ENOENT: no such file or directory, open '/tmp/x/package.json'
npm WARN x No description
npm WARN x No repository field.
npm WARN x No README data
npm WARN x No license field.

+ @ethersphere/swarm-cli@1.2.0
added 126 packages from 197 contributors and audited 126 packages in 32.527s

15 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

[nix-shell:/tmp/x]$ ./node_modules/.bin/swarm-cli --help
Swarm CLI 1.2.0 - Manage your Bee node and interact with the Swarm network via the CLI

█ Usage:

swarm-cli COMMAND [OPTIONS]

█ Available Groups:
```

Reply:
you are right, my bad, accidentaly I had an npm project in my home folder. 
But therefore, if some parent folder has `package.json` this line won't work - as in my case.